tests/e2e/annotation-change-collector/chainsaw-test.yaml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: annotation-change-collector
+spec:
+  steps:
+    - name: step-00
+      description: collectors with an extra annotation
+      try:
+        - apply:
+            file: 00-install-collectors-with-extra-annotation.yaml
+        # deployment
+        - assert:
+            file: 00-assert-deployment-with-extra-annotation.yaml
+        # daemonset
+        - assert:
+            file: 00-assert-daemonset-with-extra-annotation.yaml
+        # statefulset
+        - assert:
+            file: 00-assert-statefulset-with-extra-annotation.yaml
+
+    - name: step-01
+      description: collectors with changed extra annotations
+      try:
+        - update:
+            file: 01-install-collectors-with-annotation-change.yaml
+        # deployment
+        - assert:
+            file: 01-assert-deployment-with-annotation-change.yaml
+        # daemonset
+        - assert:
+            file: 01-assert-daemonset-with-annotation-change.yaml
+        # statefulset
+        - assert:
+            file: 01-assert-statefulset-with-annotation-change.yaml
+
+    - name: step-02
+      description: delete extra annotation from collector
```

Comment:
can you change this test to ensure that externally added annotations/labels to the deployment/statefulset/daemonset resources are not removed by the operator? I believe those are the only things we want to ensure are _always_ propagated.

Reply:
> can you change this test to ensure that externally added annotations/labels to the deployment/statefulset/daemonset resources are not removed by the operator?
By externally added, you mean manually/directly provided for the deployment/statefulset/daemonset?

Reply:
yeah, the thought is that a user may have another operator installed which is adding labels/annotations outside of our loop.

Reply:
Done.
Now the `annotation-change-collector` and `label-change-collector` e2e tests have a manual annotation/label step and we assert their existence even after we remove the collector CR's annotations and labels.